languages/JavaScript/exercises/concept/arrays-basic/.docs/introduction.md

```diff
@@ -0,0 +1,8 @@
+In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or ojects, even mixed types. The array elements can be accesed by their index.
+
+Examples of equivalent declaration:
+
+```javascript
+const numbers = [1, 'two', 3, 'four']; // most common`
+const numbers = new Array(1, 'two', 3, 'four');
```

Comment:
It might be useful to _not_ explain the `Array` notation here but just include it in the `after.md` document. I don't think the student needs to know about the `Array` notation to solve the exercise, right?